app/views/home/_project-group.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="mb-8">
+  <div class="flex mb-4 px-4">
+    <div class="w-16 content-center"><%= inline_svg_pack_tag "media/svgs/group-#{group[:name].downcase}.svg" %></div>
+    <div class="w-full">
+      <div><%= section_name group[:name] %></div>
+      <div class="w-full flex flex-col sm:flex-row justify-between flex-shrink-0">
+        <div class="flex-1">
+          <%= group[:body] %>
+        </div>
+        <%= link_to projects_path(project_types: group[:project_types].join(',')), class: 'float-right text-indigo-700 font-bold group' do %>
+          See all <%= group[:projects_count] %> <%= 'project'.pluralize group[:projects_count] %> <%= inline_svg_pack_tag 'media/svgs/chevron-right.svg', class: 'inline-block fill-current transform transition-transform duration-100 -translate-x-1 group-hover:translate-x-0' %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+
+  <div class="w-full px-4 sm:px-0 space-y-bottom-4 sm:grid grid-cols-2 lg:grid-cols-3 sm:gap-6">
```

Comment:
Should this be
```suggestion
  <%= list_cards do %>
```
like in _project-category.html.erb? `list_cards` seems to expand to almost the same html as what's here.